packages/x-tailwindcss/demo/src/components/xds-layout-utils.vue

```diff
@@ -97,12 +120,23 @@
     })
     public expand!: string[];
 
+    public modalContent = {
+      'No margin': this.noMargin.map(addParentClasses(this.base)),
+      'On margin': this.onMargin,
+      Overlap: this.overlap.map(addParentClasses(this.base)),
+      Expand: this.expand.map(addParentClasses(this.base))
+    };
+
+    public sectionDescriptions = {
+      'No margin': 'Removes the margin from one or both sides of the layout item.',
+      'On margin': 'Positions an element in one of the side margins of the layout.',
+      Overlap: 'Positions an element over a layout item.',
+      Expand: 'Makes a layout item to fit the container height.'
+    };
+
     protected get sections(): ShowcaseSections {
       return {
-        'No margin': this.noMargin.map(addParentClasses(this.base)),
-        'On margin': this.onMargin,
-        Overlap: this.overlap.map(addParentClasses(this.base)),
-        Expand: this.expand.map(addParentClasses(this.base))
+        '': [this.base]
```

Comment:
Why not use `Default` here instead of `''`?

Reply:
Using `Default` there will print the text next to the button to open the modal, I didn't see it really necessary

![image](https://user-images.githubusercontent.com/77450928/228451270-28436efc-e2d6-40cc-946b-7809ecfb628f.png)